kover-gradle-plugin/build.gradle.kts

```diff
@@ -87,7 +108,7 @@ val functionalTest by tasks.registering(Test::class) {
         systemProperties["junit.jupiter.execution.parallel.config.fixed.parallelism"] =
             junitParallelism?.toIntOrNull()?.toString() ?: "2"
         // this is necessary if tests are run for debugging, in this case it is more difficult to stop at the test you need when they are executed in parallel and you are not sure on which test the execution will pause
-        systemProperties["junit.jupiter.execution.parallel.enabled"] = if (junitParallelism == "no") "false" else "true"
+        systemProperties["junit.jupiter.execution.parallel.enabled"] = if (junitParallelism == "no") "false" else "false"
```

Comment:
![false-false](https://github.com/Kotlin/kotlinx-kover/assets/13045611/05aec770-81b0-4dd8-9286-cbfd4d1ce623)

Reply:
`#define true false`

Reply:
Jokes aside, why parallel execution is now always disabled?

Reply:
I forgot to change it back, it makes it easier to connect the debugger.